py/torch_tensorrt/dynamo/runtime/_PythonTorchTensorRTModule.py

```diff
@@ -121,6 +124,52 @@ def setup_engine(self) -> None:
         self.engine = runtime.deserialize_cuda_engine(self.serialized_engine)
         self.context = self.engine.create_execution_context()
 
+        if self.settings.strip_engine_weights:
+            assert (
+                self.settings.make_refittable
+            ), "weight-stripped engines must be refittable, please set make_refittable=True"
+
+            # Refit the weights
+            refitter = trt.Refitter(self.engine, TRT_LOGGER)
```

Comment:
Can you use this function?
https://github.com/pytorch/TensorRT/blob/fa02fd3e3a85a6042d11a00cd386f6b69c1d6c4b/py/torch_tensorrt/dynamo/_refit.py#L138

Reply:
The function requires `input_list` which is not provided in the caller.